src/components/hotel/index.js

```diff
@@ -30,7 +30,7 @@ class HotelPage extends Component {
     for (var key in amenitiesObject) {
       if (!amenitiesObject.hasOwnProperty(key)) continue;
       if (key === fact) {
-        if (amenitiesObject[key] === typeof boolean || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
+        if (typeof amenitiesObject[key] === 'boolean' || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
```

Comment:
I meant in addition to the `=== 'true'` and `=== 'false'` clauses, not in place of the entire condition. So the specific `'true'` and `'false'` values would still be matched.

i.e. 

```javascript
if (typeof amenitiesObject[key] !== 'string' || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
...
}
```